src/main/scala/edu/ie3/simona/agent/participant/ParticipantAgentFundamentals.scala

```diff
@@ -1725,4 +1760,83 @@ case object ParticipantAgentFundamentals {
 
     ApparentPower(p, q)
   }
+
+  /** Determine the average apparent power within the given tick window
+    *
+    * @param tickToResults
+    *   Mapping from data tick to actual data
+    * @param windowStart
+    *   First, included tick of the time window
+    * @param windowEnd
+    *   Last, included tick of the time window
+    * @param activeToReactivePowerFuncOpt
+    *   An Option on a function, that transfers the active into reactive power
+    * @return
+    *   The averaged apparent power
+    */
+  def averageApparentPowerAndHeat(
```

Comment:
Here, we could re-use `averageApparentPower` instead of duplicating all this code.